File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Data/Domain/Collaboration/CollaborationDraft.vb
Imports VecompSoftware.DocSuiteWeb.Data

<Serializable()>
Public Class CollaborationDraft
    Inherits AuditableDomainObject(Of Guid)

#Region " Fields "
#End Region

#Region " Properties "
    Public Overridable Property Incremental As Integer

    Public Overridable Property IsActive As Short

    Public Overridable Property Description As String

    Public Overridable Property Data As String

    Public Overridable Property DraftType As Integer

    Public Overridable Property Collaboration As Collaboration
    Public Overridable Property IdDocumentUnit As Guid?
#End Region

#Region "Constructor"
    Public Sub New()

    End Sub
#End Region

End Class

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Entity/Collaborations/CollaborationDraft.cs
using System;
using VecompSoftware.DocSuiteWeb.Entity.Collaborations;
using VecompSoftware.DocSuiteWeb.Entity.DocumentUnits;

namespace VecompSoftware.DocSuiteWeb.Entity.Collaborations
{
    public class CollaborationDraft : DSWBaseEntity
    {
        #region [ Constructor ]
        public CollaborationDraft() : this(Guid.NewGuid()) { }

        public CollaborationDraft(Guid uniqueId)
            : base(uniqueId)
        {
        }
        #endregion

        #region [ Properties ]
        public virtual bool IsActive { get; set; }

        public virtual string Description { get; set; }

        public virtual string Data { get; set; }

        public virtual int DraftType { get; set; }
        #endregion

        #region [ Navigation Properties ]
        public virtual Collaboration Collaboration { get; set; }
        public virtual DocumentUnit DocumentUnit { get; set; }
        #endregion
    }
}
